groups/instance/variables.tf

```
@@ -136,65 +132,54 @@ variable "alb_deregistration_delay" {
   type        = number
 }
 
-variable "asg_health_check_grace_period" {
-  description = "Time (in seconds) after instance comes into service before checking health. Default 300"
-  default     = 150
+variable "asg_default_cooldown" {
+  default     = 10
+  description = "The amount of time, in seconds, after a scaling activity completes before another scaling activity can start"
   type        = number
 }
 
 variable "asg_desired_capacity" {
-  description = "The number of Amazon EC2 instances that should be running in the group."
   default     = 1
+  description = "The number of Amazon EC2 instances that should be running in the group."
   type        = number
 }
 
+variable "asg_health_check_grace_period" {
+  default     = 150
+  description = "Time (in seconds) after instance comes into service before checking health. Default 300"
```

There's a mismatch here between the default value and the value declared in the description:

```suggestion
  description = "Time (in seconds) after instance comes into service before checking health. Default 150"
```